hack/ci/e2e.sh

```diff
@@ -106,6 +106,9 @@ apiVersion: kind.sigs.k8s.io/v1alpha3
 nodes:
 # the control plane node
 - role: control-plane
+  extraMounts:
+  - containerPath: /etc/audit-policy.yaml
+  - hostPath: ${ARTIFACTS}/audit-policy.yaml
```

Comment:
Where should I place the audit-policy.yaml... a heredoc in the first line of create_cluster?

Reply:
path in the container that is passed to the api server ideally should be:
`/etc/kubernetes/audit-policy.yaml`

> Where should I place the audit-policy.yaml... a heredoc in the first line of create_cluster?

do you mean from where to mount the policy?

Reply:
the `${ARTIFACTS}` location AFAIK is only for artifacts that are the result of the test.
i think one option is to download the policy under `/tmp/` and mount from there.